tool/lib/commands/fix_goldens.dart

```diff
@@ -0,0 +1,101 @@
+// #!/bin/bash -e
```

Comment:
can we get rid of these? also add copyright please

Reply:
yep this was accidentally left behind